Examples/Scripts/Python/itk_seeding.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python3
+import os
+import argparse
+
+import acts
+import acts.examples
+
+from acts.examples import CsvSpacePointReader
+
+u = acts.UnitConstants
+
+
+def runITkSeeding(field, outputDir, s=None):
+
+    # Read input space points from input csv files
+    evReader = CsvSpacePointReader(
+        level=acts.logging.INFO,
+        inputStem="spacepoints",
+        inputCollection="pixel",
+        inputDir="acts/Examples/Scripts/Python/data_itk_seeding/CsvSpacePointsOutput_singleMu_100GeV_300evnts/",
```

Comment:
Consider using a temporary file. If you want to try this, i can provide a snippet.

Reply:
yes, please send me the snippet